### ISSUE TITLE
[CLI]:  Image not getting logged when restart and offline mode

### ISSUE THREAD

**Issue by nicolas-dufour** [wandb/client#3696]
### Describe the bug

I'm running experiments on a cluster with offline mode.  At the end of the training, i log some images on a test set. When i do not have to restart the training, everything works fine. After restarting, however, the images aren't logged any more. 
I'm using wandb 0.12.4 and pytorch lightning 1.6.0

I've inspected the log folder and images are indeed logged in the wandb folder, but they don't seem to be uploaded when syncying. 

No error message is prompted

### Environment

WandB version: 0.12.4

OS: Linux

Python version: 3.9

Versions of relevant libraries:
Pytorch lightning 1.6.0

**Comment by ramit-wandb** [comment]
Hi @nicolas-dufour,

I'm sorry this is happening to you. Since you mentioned you do see these runs in your `wandb` folder, could you try running `wandb sync path/to/run/folder` as a first step at debugging this and checking if the images show up as expected in the run?

Additionally, it would be a great help if you could send me a link to a affected run, the associated `debug.log` and `debug-internal.log` file and the output of a command like `tree` in your wandb folder for an affected run. These will help me look into this on my end.

Thanks,
Ramit

**Comment by ramit-wandb** [comment]
​Hi @nicolas-dufour,

We wanted to follow up with you regarding your support request as we have not heard back from you. Please let us know if we can be of further assistance or if your issue has been resolved.

Best,
Weights & Biases
​

**Comment by ramit-wandb** [comment]
​Hi @nicolas-dufour, since we have not heard back from you we are going to close this request. If you would like to re-open the conversation, please let us know!
